doc/news/_preparation_next_release.md

```diff
@@ -276,6 +276,16 @@ The following section lists news about the [modules](https://www.libelektra.org/
 - [mmapstorage](https://www.libelektra.org/plugins/mmapstorage) is now able to persist the Global KeySet, which is used by the `cache` plugin. _(Mihael Pranjić)_
 - Fixed support for `kdb import` and `kdb export`. _(Mihael Pranjić)_
 
+### semlock
+
+removed due to:
+
+- constant pain
```

Comment:
Was it because of the problems in hooks, because of OS-specific problems, or something else?

Reply:
Manly OS-specifics `/dev/shm should be mounted as tempfs` for example. But I also see no reason for the semlock, the main reason was the race test, but the timestamp consistency check between KdbGet and KdnSet made the race test broken by design.

Reply:
Yes, you are right. The semlock plugin alone cannot fix the race tests.